torch/profiler/profiler.py

```diff
@@ -45,6 +45,33 @@ def _default_schedule_fn(_: int) -> ProfilerAction:
     """
     return ProfilerAction.RECORD
 
+def tensorboard_trace_handler(dir_name: str, worker_name: Optional[str] = None):
+    """
+    Outputs tracing files to directory of ``dir_name``, then that directory can be
+    directly delivered to tensorboard as logdir.
+    ``worker_name`` is used to distinguish different workers in distributed scenario,
+    it will be set to '[hostname]_[pid]' by default.
+
+    .. note::
+        Example of using Tensorboard plugin for pytorch:
+
+        - pip install -i https://test.pypi.org/simple/ tensorboard-plugin-torch-profiler==0.1.0
+        - tensorboard --logdir=``dir_name``
+    """
+    def handler_fn(prof) -> None:
+        import os, time, socket
+        nonlocal worker_name
+        if not os.path.isdir(dir_name):
+            try:
+                os.makedirs(dir_name)
```

Comment:
add `exist_ok = True`?
e.g. in distributed training case when a shared storage is used, it is possible that multiple workers may try to create dir at the same time.